app/views/home/prices.html.erb

```diff
@@ -0,0 +1,48 @@
+<main class="container-fluid">
+
+ <section class="prices-head">
+    <div class="prices-text">
+        <h2> <%= t('prices_text_head') %></h2>
+            <p><%= t('prices_text2') %></p> 
```

Comment:
This `p` should be at the same level of indentation as the `h2` above. The same applies for other `p`s in this file.